examples/workflows/go.mod

```diff
@@ -0,0 +1,11 @@
+module workflows
+
+go 1.16
+
+replace github.com/gravitational/teleport/api => ../../api
```

Comment:
Most people who follow this example won't be putting the code in-tree. Can we do without the replace directive here?